.gitlab-ci.yml

```
@@ -0,0 +1,21 @@
+stages:
+  - build
+  - post_build
+
+before_script:
+  - echo "STARTING BEFORE_SCRIPT"
+  - apt-get update
+  - apt-get -y install python-pip python-dev build-essential
+  - pip install awscli --upgrade
+  - test $SPEL_CI == "true" && export SPEL_VERSION=$(date +%Y.%m.dev%s)
```

This line will exit non-zero if the test fails... Does the non-zero exit cause the build to halt and exit? If so, may need to use the same inline if block as in buildspec.yml, or force it to exit zero...

```suggestion
  - |
    if [ "${SPEL_CI}" = "true" ]
    then
      SPEL_VERSION=$(date +%Y.%m.dev%s)
      export SPEL_VERSION
    fi
```

```suggestion
  - test $SPEL_CI == "true" && export SPEL_VERSION=$(date +%Y.%m.dev%s) || true
```